client/.eslintrc.json

```diff
@@ -25,11 +26,55 @@
       "react/no-string-refs": [ "off" ],
       "react/display-name": [ "off" ],
       "react/no-find-dom-node": [ "off" ],
-      "react/no-children-prop": [ "off" ]
+      "react/no-children-prop": [ "off" ],
+    "import/order": [
+      "warn", { 
+        "newlines-between": "always-and-inside-groups",
+        "alphabetize": {
+          "order": "asc",
+          "caseInsensitive": true 
+        },
+        "groups": [
+          "builtin",
+          "external",
+          "internal",
+          "parent",
+          ["sibling", "index"]
+        ],
+        // We can group imports further by adding rules here, the order they're defined in breaks ties when group and position are equal
+        "pathGroups":[
+          {
+            "pattern": "./*.yaml",
+            "group": "index",
+            "position": "after"
+          },
+          {
+            "pattern": "components/**/*.js",
+            "group": "external",
```

Comment:
Not sure if I'm not understanding right, but wouldn't these groups be "internal"? 
https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/order.md#importorder-enforce-a-convention-in-module-import-order

Reply:
They are, but this rule forces them to be placed in between external and internal. 

I think `group:internal, position: before` would achieve the same thing, if that's clearer we can do that 🤷